server/src/routes/auth/index.ts

```diff
@@ -0,0 +1,10 @@
+import { Router } from 'express';
+
+import { signin } from 'controllers/auth';
+import { signInValidateion } from 'validation/auth';
+
+const router = Router();
+
+router.post('/', signInValidateion, signin);
```

Comment:
```suggestion
import { signin } from 'controllers/auth';
import { signInValidation } from 'validation/auth';

const router = Router();

router.post('/', signInValidation, signin);
```

오타가 있네용 !

함수명도 `signin` -> `signIn`으로 하는건 어떤가요? 밑에는 `signInValidation`이라고 되어 있어서 !!

Reply:
수정할게요